src/Shopware/AbstractUpdatePluginsTask.php

```diff
@@ -93,6 +107,16 @@ protected function refreshPluginList()
         return true;
     }
 
+    /**
+     * Checks if the update should be executed as single remote command for all plugins.
+     *
+     * @return bool
+     */
+    protected function shouldUseSingleRemoteCommand()
+    {
+        return isset($this->options['single_remote_command']) ? $this->options['single_remote_command'] : false;
```

Comment:
Please parse the value of $this->options['single_remote_command'] as a boolean to ensure, that a bool value is always returned.

Reply:
thx done